spread_tests/scripts/build_termus.sh

```diff
@@ -0,0 +1,7 @@
+#! /usr/bin/env bash
+
+set -e
+
+pushd ${SPREAD_ROOT_DIR}
```

Comment:
We should double quote all vars for a cleaner code, shellcheck could help in this regard